File: Programmin Fundamentals/Homeworks and Labs/DataTypesAndVariables-Exercises/13.VowelOrDigit/VowelOrDigit.cs
using System;
using System.Linq;

public class VowelOrDigit
{
    public static void Main()
    {
        char symbol = char.Parse(Console.ReadLine().ToLower());

        char[] vowelLetter = new char[] { 'a', 'e', 'u', 'i', 'o' };

        if (char.IsDigit(symbol))
        {
            Console.WriteLine("digit");
        }
        else if (char.IsLetter(symbol) && vowelLetter.Contains(symbol))
        {
            Console.WriteLine("vowel");
        }
        else
        {
            Console.WriteLine("other");
        }
    }
}

File: Software Technologies/Java Basics - Exercises/src/VowelOrDigit.java
import java.util.Scanner;

public class VowelOrDigit {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        char [] vowels =  {'a','e','o','u','i','y'};
        char symbol =scanner.nextLine().charAt(0);

        boolean isDigit = Character.isDigit(symbol);
        boolean isVowel = false;
        for (int i = 0; i < vowels.length; i++) {
            if(symbol == vowels[i]){
                isVowel = true;
                break;
            }
        }

        if(isDigit){
            System.out.println("digit");
        }else if (isVowel){
            System.out.println("vowel");
        }else{
            System.out.println("other");
        }

    }
}
